popups/cloud-games/popup.html

```diff
@@ -10,21 +10,31 @@
     <script src="./popup.js" type="module"></script>
   </head>
 
-  <body style="display: none" v-show="loaded">
-    <div class="games" v-for="project of projectsSorted">
-      <div class="title">
-        <a class="project-name" :href="'https://scratch.mit.edu/projects/' + project.id" target="_blank"
-          >{{ project.title }}</a
-        >
-        <span class="float-right"><img class="small-icon" src="../../images/icons/users.svg" /> {{ project.amt }}</span>
-      </div>
-      <div class="project-details" v-show="true || project.amt > 0 && project.extended">
-        <div class="username-list">
-          <a v-for="user of project.users" :href="'https://scratch.mit.edu/users/' + user" target="_blank"
-            >{{ user }}</a
+  <body style="display: none" v-show="true">
```

Comment:
not sure why you need those 2 atterabutes

Reply:
hide the body until vue loads